translator/conventions/opentelemetry.go

```diff
@@ -67,6 +67,24 @@ const (
 	AttributeCloudAccount          = "cloud.account.id"
 	AttributeCloudRegion           = "cloud.region"
 	AttributeCloudZone             = "cloud.zone"
+	AttributeDeploymentEnvironment = "deployment.environment"
+	AttributeOSType                = "os.type"
+	AttributeOSDescription         = "os.description"
+)
+
+// OpenTelemetry Semantic Convention values for Resource attribute "telemetry.sdk.language" values.
+// See: https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions/README.md
+const (
+	SDKLangCPP    = "cpp"
```

Comment:
Can we prefix by `Attribute` like all other conventions are?

Reply:
These are enumerated values of a particular attribute. I changed the names to clarify.